python/pyspark/mllib/tests.py

```diff
@@ -75,22 +75,23 @@
     # No SciPy, but that's okay, we'll skip those tests
     pass
 
-ser = PickleSerializer()
```

Comment:
I recall seeing some concurrency issues with ser too (it has been a while). I'm pushing up a rebase/update.